src/index.js

```diff
@@ -178,14 +210,59 @@ const moduleToApp = (ddoc, module) =>
     digest: ddoc._attachments[module].digest,
   });
 
+const deployDdoc = stagedDdoc => {
+  info(`Deploy staged ddoc ${stagedDdoc._id} to ${DDOC}`);
+
+  const deletedStagedDdoc = {
+    _id: stagedDdoc._id,
+    _rev: stagedDdoc._rev,
+    _deleted: true
+  };
+
+  info('Getting currently live DDOC');
+  return db.get(DDOC)
+    .catch(err => {
+      if (err.status !== 404) {
+        throw err;
+      }
+
+      info('No existing live DDOC');
```

Comment:
I think it's OK as an info? To me warns are unexpected or surprising things that should not be (like errors) but we can continue on with (unlike errors).

Ideally we'd warn on an upgrade and not even mention it on a fresh deploy (or mention it if there *is* an existing ddoc).